### PR TITLE
Nerfs the GL-70 grenade launcher's fire rate and the grenade's explosion delay.

### PR DESCRIPTION
**Pull request by Arturlang**
## About The Pull Request
Before you say it. This is 100% a xope PR.
So, usually if someone throws a grenade, there's plenty of ways to counter it, banishing as wraith, or using a fling ability on it to throw it back, unless it's cooked perfectly which is risky for the marine, you have enough time to react, what grenade launchers do is let you shoot 6 grenades, each at a delay of 1.4 second, which is constant, oppresive pain to xenos and incredibly hard to counter without having a crusher, which even then can just get flung around and be unable to do anything due to the explosive stagger slowdown, this PR makes grenade launcher's grenades explode in 1.2 seconds, whereas it used to be in one second, and makes the multinade grenade launcher's fire delay 1.4 seconds as well. You can still shoot grenades near-constantly but now xenos will have 1.2 second of a delay to actually react to the grenade.
## Why It's Good For The Game
Xenos will no longer need godly reaction speeds to counter grenade launchers.
## Changelog
:cl:
balance: GL-70 Grenade launcher's grenades now explode in 12 ticks/1.2 seconds.
balance: GL-70 Grenade launcher's fire rate is now one shot per 14 ticks/1.4 seconds.
/:cl:
TL:DR
Grenades shot from a multi GL blow up 20% slower, 1.2 second delay, and it's fire rate has been changed to 1.4

### CODE REVIEW

**Comment by Tommy-211** [comment]
Sound the retard alerts, someone broke out of a mad house and posted this pr

**Comment by Nuker555** [comment]
This makes gl near useless.

**Comment by Bestparty** [comment]
nade

**Comment by Arturlang** [comment]
If you want to know why this PR was made, play against a grenade launcher as a caste that's slow.

> This makes gl near useless.

You might have to lead xenos a little more yes, but uncounterable grenade spam is not a good thing. Keep in mind it still blows up *really* damn fast.

**Comment by Barnet2** [comment]
I still think the issue lies in the **600 free HEDP nades** in the vendor, but this could work. Reducing the amount of nades just means one person is going to horde them, anyway. You can also scope the GL still I believe? Unless that was removed.

**Comment by Arturlang** [comment]
> I still think the issue lies in the **600 free HEDP nades** in the vendor, but this could work. Reducing the amount of nades just means one person is going to horde them, anyway. You can also scope the GL still I believe? Unless that was removed.

You can miniscope it yes, but it has a max range of 15

**Comment by Ragecrabbo** [comment]
Nade timer changed from 1 second to 1.5 seconds? Yup its over nade launcher is useless pack it up boys

**Comment by usnpeepoo** [comment]
merge this pr now 🔥

**Comment by Pariah919** [comment]
Other than being admitted salt, why are you flat nerfing every single grenade launcher in the game at least make it modifeable.

**Comment by Arturlang** [comment]
> Other than being admitted salt, why are you flat nerfing every single grenade launcher in the game for this, at least make it modifeable.

Will do

**Comment by Tommy-211** [comment]
This is literally a salt nerf how is this even being considered

**Comment by Tommy-211** [comment]
we should make warlock lance wind up 10 seconds just because we die to it sometimes

**Comment by Lantekk** [comment]
Lot of Great PR's lately.

**Comment by Tommy-211** [comment]
Also increasing the GL's fire rate will just invalidate GLs, people can throw grenades much faster than 1.5 seconds, maybe take a break from the game from a bit rather than trying to change things in the game because you lost to it.
manchild

**Comment by Arturlang** [comment]
Could remove the extra grenade launcher delay so you can have multiple grenades in the field, maybe?

**Comment by coldironwarrior** [comment]
so the high-risk shipside hive plan of yours didn't work because someone GL'd you and you want to nerf it to make it equal to hand-throwing...?

**Comment by Tommy-211** [comment]
> Could remove the extra grenade launcher delay so you can have multiple grenades in the field, maybe?

Could close the xope pr maybe? 
There is no need for this pr, grow a pair.

**Comment by Tommy-211** [comment]
Don't go to prc talking about how I'm "mad" you are literally making a PR because you died to something - and you admit that it is a salt pr.

**Comment by Kuro020** [comment]
> > Could remove the extra grenade launcher delay so you can have multiple grenades in the field, maybe?
> 
> Could close the xope pr maybe? There is no need for this pr, grow a pair.

How to get gitbanned 101
Don't be an asshole in pr comments

**Comment by chizzy376** [comment]
Grenade launcher is soooo overpowered right now, look at all these crosses it's making! Way too op of a xeno weapon, this nerf should help marines prevent that much ff from being dispensed
grenade luncher

**Comment by Arturlang** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/assets/24881678/a9c833e2-02d0-4149-9671-4660c73c8509

This is what the fire rate looks like with these changes

**Comment by Arturlang** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/assets/24881678/9baaec60-6651-403b-9ecf-5b2309fab9fb

This is before this PR.

**Comment by Arturlang** [comment]
> so the high-risk shipside hive plan of yours didn't work because someone GL'd you and you want to nerf it to make it equal to hand-throwing...?
Keep in mind if you want something to blow up as fast as the GL, you're going have to hold it for ~3 seconds and then throw it, which incurs some risk and warning for any xeno doing it. The GL will be able to shoot 2-3 grenades during that time, while if you were hand-throwing it you'd still be warming up your second.

**Comment by Strayed-Raven** [comment]
When everything is reduced, mitigated, and gutted: There will be no true choice. Including after this with grenade launchers.

**Comment by chizzy376** [comment]
> 2023-11-16.23-51-04.mp4 
> This is what the fire rate looks like with these changes. I debated increasing the delay so there would be actual breaks in the firing, but I was unsure if it was wise given the discussions

alright nvm this literally makes them useless please don't merge

**Comment by Arturlang** [comment]
> alright nvm this literally makes them useless please don't merge

Would you kindly explain why? Atleasst to me the difference in delay is awfully smal

**Comment by srxtreme** [comment]
> > alright nvm this literally makes them useless please don't merge
> 
> Would you kindly explain why? Atleasst to me the difference in delay is awfully small, you might sometimes need to lead the grenade or directly hit the xeno, you just won't be able to consistently get xenos if you start out on the outer rage.

This will ensure a near zero hitrate on faster castes in any scenario (as opposed to near zero hitrate in open spaces), and a massively reduced hitrate on any xeno capable of moving even at moderate speeds. it'd also kill the ability for GL to solokill non bald xenos.

**Comment by ilikesaying** [comment]
if you're going to nerf the GL's attack stats at least give it increased storage to compensate.

**Comment by DeltaFire15** [comment]
>[...]without having a crusher, which even then can just get flung around and be unable to do anything due to the explosive stagger slowdown[...]

This is not true. Crushers do not get staggered or slowed by explosives, provided they do not engage while heavily sundered (which you shouldn't). The displacement still does apply, though.

**Comment by Arturlang** [comment]
> > [...]without having a crusher, which even then can just get flung around and be unable to do anything due to the explosive stagger slowdown[...]
> 
> This is not true. Crushers do not get staggered or slowed by explosives, provided they do not engage while heavily sundered (which you shouldn't). The displacement does still apply, though.

Good point, but crushers are still very slow, and it's very easy to get sundered as a result, especially if nobody is willing to play a support caste

**Comment by Lumipharon** [comment]
I don't really think the grenade launcher has this much of an issue. Is having 600 nades for free making gl's easily spammable on the other hand? Maybe.
